core/templates/dev/head/pages/exploration_player/PlayerTranscriptService.js

```diff
@@ -119,7 +130,7 @@ oppia.factory('playerTranscriptService', ['$log', function($log) {
       return this.getCard(transcript.length - 1);
     },
     getNumSubmitsForLastCard: function() {
-      return this.getLastCard().answerFeedbackPairs.length;
+      return numOfAnswersSubmitted;
```

Comment:
Just to check, does this get reset at each new card or at the start of each exploration?